frontend/public/components/utils/_skeleton-screen.scss

```diff
@@ -42,7 +42,7 @@ $skeleton-overview-title-position: 50px $skeleton-overview-tile-padding;
 $skeleton-overview-title-bone: linear-gradient(white $skeleton-overview-title-height, transparent 0);
 $skeleton-overview-title-width: 38%;
 
-$skeleton-tile-height: 290px; // height of catalog tiles
+$skeleton-tile-height: 240px; // height of catalog tiles
```

Comment:
The height of the skeleton tile does not match the height of the cards because the cards can have different heights (see earlier comment).

Reply:
Yeah, this was an issue. We obviously don't have data yet at this point to make them match the real tiles, which are based on the size of the data. I sized them to match the top few so it wouldn't be jarring, but if you have thoughts on a better approach, let us know.

Reply:
Can use some `var + var` incantation instead of hard-coded `px`?